client-mvp-04/src/store/configureStore.js

```diff
@@ -0,0 +1,15 @@
+  
+import { createStore, combineReducers } from 'redux';
+import authReducer from '../reducers/authReducer';
+
+export default () => {
+    // Store creation
+    const store = createStore(
+        combineReducers({
+            auth: authReducer,
+        }), /* preloadedState, */
+        window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
+    );
+    
+    return store;
+}
```

Comment:
please add an empty line at the end of the file
https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file